client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -11,19 +11,24 @@ import { flowRight as compose, includes } from 'lodash';
  * Internal dependencies
  */
 import PluginIcon from 'calypso/my-sites/plugins/plugin-icon/plugin-icon';
-import { Button } from '@automattic/components';
+import { Button, Card } from '@automattic/components';
 import Rating from 'calypso/components/rating';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 import { getSitesWithPlugin } from 'calypso/state/plugins/installed/selectors';
+import { withLocalizedMoment } from 'calypso/components/localized-moment';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const PREINSTALLED_PLUGINS = [ 'Jetpack by WordPress.com', 'Akismet', 'VaultPress' ];
+const PREINSTALLED_PLUGINS = [
```

Comment:
Do these plugins have an ID value or something like that? Tracking by the name seems like it could be fragile potentially.